pkg/bot/interactive/help.go

```diff
@@ -209,10 +209,11 @@ func (h *HelpMessage) aiPlugin() []api.Section {
 		{
 			Base: api.Base{
 				Header:      "🤖 AI powered Kubernetes assistant",
-				Description: fmt.Sprintf("`%s ai` use natural language to ask any questions", api.MessageBotNamePlaceholder),
+				Description: fmt.Sprintf("`%s ai` use natural language to ask any questions\n`%s ai scan` perform a cluster-wide scan for issues", api.MessageBotNamePlaceholder, api.MessageBotNamePlaceholder),
 			},
 			Buttons: []api.Button{
 				h.btnBuilder.ForCommandWithoutDesc("Ask a question", "ai hi!", api.ButtonStylePrimary),
+				h.btnBuilder.ForCommandWithoutDesc("Cluster Scan", "ai scan", api.ButtonStylePrimary),
```

Comment:
Please double check with @miklosp if we should have these two lines. IMO the `Ask a question` button is quite unnecessary and as now we have a very useful command, we could keep a single button.

Reply:
@miklosp please advise.

Reply:
![image](https://github.com/kubeshop/botkube/assets/1308018/f60468d3-597a-4071-b88c-b64b0ff393e8)
updated with suggested changes.